server/routes/seriesRoutes.test.js

```diff
@@ -88,3 +88,61 @@ describe('Given a /series/ route', () => {
     });
   });
 });
+
+describe('Given a /series/:idSerie route', () => {
+  describe('When it receives a delete request with a serie id', () => {
+    test('Then it should respond with the deleted serie', async () => {
+      const response = await request.delete(`/series/${newSerie1.id}`).expect(200);
```

Comment:
better add a empty line betweeen arrange - act - space, it will be more easy to read the code